### PR TITLE
Add nuts-node:dev build

### PR DESCRIPTION
**Pull request by gerardsn**
This creates a new docker image that is build on top of the nuts-node and adds node-exporter to analyze performance

### CODE REVIEW

File: development/performance_analyzer/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+This directory contains all files needed to build a docker container that includes [node-exporter](https://github.com/prometheus/node_exporter) for performance testing.
+
+## Configuring node-exporter
+Many [collectors](https://github.com/prometheus/node_exporter#collectors) are configured by default. 
+Changes to the default settings can be made in `node-exporter.conf` (collectors, port, ...).
+
+If a specific setup is needed for a test, just create a new configuration file and mount it to `/etc/conf.d/node-exporter`
```
Comment (woutslakhorst) [inline development/performance_analyzer/README.md:7]: Isn't this a rare exception?

File: development/performance_analyzer/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    metrics_path: '/metrics'
+    scrape_interval: 5s
+    static_configs:
+      - targets: 
+        - '<address>:1323'  # metrics exposed on nuts-node/metrics
+        - '<address>:9100'  # metrics exposed by node-exporter
+```
+
+Note: metrics prepended with `go_` from the `node-exporter` and `nuts-node` targets are different. 
+Perhaps these relate to the target process (nuts-node / node-exporter) itself??
```
Comment (woutslakhorst) [inline development/performance_analyzer/README.md:34]: Yes, so the GO stuff from the node-exporter should be ignored.

**Review by woutslakhorst — COMMENTED** [review]
Some comments

File: development/performance_analyzer/README.md (hunk 2 above)
Comment (gerardsn) [reply development/performance_analyzer/README.md]: updated

**Review by woutslakhorst — APPROVED** [review]

**Pull request merged**